concent_api/concent_api/settings/development.py

```diff
@@ -27,3 +28,9 @@
 STORAGE_CLUSTER_ADDRESS = 'http://devel.concent.golem.network/'
 
 GETH_CONTAINER_ADDRESS = 'http://localhost:8545'
+
+STORAGE_SERVER_INTERNAL_ADDRESS = 'http://devel.concent.golem.network/'
```

Comment:
Please don't use the address of the cluster. If won't work and if it would, it would not do us any good. This should probably be localhost or a container name or something. If anyone's going to have the storage server running locally it will be in a docker container.

How do you run the server locally? The development settings are meant to work out of the box with that.

Reply:
Outdated, already done.